packages/bitcore-node/test/integration/wallet-benchmark.integration.ts

```diff
@@ -282,13 +282,13 @@ describe('Wallet Benchmark', function() {
       expect(largeAddressBatch.length).to.deep.equal(1000);
 
       console.log('Checking');
-      const importedWallet1 = await createWallet(smallAddressBatch, 0, network);
-      const importedWallet2 = await createWallet(mediumAddressBatch, 1, network);
-      const importedWallet3 = await createWallet(largeAddressBatch, 2, network);
+      const importedWallet4 = await createWallet(smallAddressBatch, 4, network);
```

Comment:
These shouldn't need to be changed, createWallet loads the wallet or creates it if it doesn't exist. This should just load the previously created wallet.